ern-api-gen/src/generateProject.js

```diff
@@ -92,14 +93,20 @@ export function generatePackageJson ({
   }, null, 2)
 }
 
-export function generateInitialSchema ({
+export async function generateInitialSchema ({
   namespace,
-  shouldGenerateBlankApi
+  shouldGenerateBlankApi,
```

Comment:
I thought you were getting rid of this ?

Reply:
@belemaire yes there is another PR i am testing and updating the test